glean-core/ios/Glean/Glean.swift

```diff
@@ -162,6 +162,13 @@ public class Glean {
 
                 if !enabled {
                     Dispatchers.shared.cancelBackgroundTasks()
+                    // If uploading is disabled, we need to send the deletion_request ping
+                    Dispatchers.shared.launchConcurrent {
```

Comment:
Can you file a bug for adding the 'deletion_request' to Python?

Reply:
Well, for the 'upload' part, since the creation itself happens in Rust.

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1600259